articles/quickstarts/index.md

```diff
@@ -19,17 +19,150 @@ Learn how to install the Microsoft Quantum Development Kit (QDK), so that you ca
+
+### Configure for Jupyter Notebooks and Python
+
+Jupyter Notebooks allow running in-place code alongside instructions, notes, and other content. This environment is ideal for writing Q# code with embedded explanations or quantum computing interactive tutorials. 
+
+IQ# (pronounced i-q-sharp) is an extension to the .NET Core SDK, primarily used by Jupyter and Python, that provides the core functionality for compiling and simulating Q# operations.
+
+Here's what you need to do to start creating your own Q# notebooks.
+
+### [Install using conda (recommended)](#tab/tabid-conda)
+
+1. Install [Miniconda](https://docs.conda.io/en/latest/miniconda.html) or [Anaconda](https://www.anaconda.com/products/individual#Downloads). **Note:** 64-bit installation required.
+
+1. On Windows, open an Anaconda Prompt, or on macOS or Linux, open a terminal window.
+   - Or, if you prefer to use PowerShell or pwsh: open a shell, run `conda init powershell`, then close and re-open the shell.
+1. Create and activate a new conda environment named `qsharp-env` with the required packages (including Jupyter Notebook and IQ#) by running the following commands:
+    ```
+    conda create -n qsharp-env -c quantum-engineering qsharp notebook
+    conda activate qsharp-env
+    ```
+1. Run `python -c "import qsharp"` from the same terminal to verify your installation and populate your local package cache with all required QDK components.
+
+To start writing Q# code with Python, see [Develop with Q# and Python](xref:microsoft.quantum.install.python).
 
-To proceed with installing the QDK and creating Q# projects, select your preferred setup:
+### [Install using .NET CLI (advanced)](#tab/tabid-dotnetcli)
 
-[Develop with Q# applications](xref:microsoft.quantum.install.standalone) - Choose this approach to work with Q# from the command prompt. This does not require a driver or a host program like the below options.
+1. Prerequisites:
 
-[Develop with Q# Jupyter Notebooks](xref:microsoft.quantum.install.jupyter) - Select this environment to run Q# code in cells with embedded text or create quantum computing interactive tutorials. 
+    - [Python](https://www.python.org/downloads/) 3.6 or later
+    - [Jupyter Notebook](https://jupyter.readthedocs.io/en/latest/install.html)
+    - [.NET Core SDK 3.1](https://dotnet.microsoft.com/download/dotnet-core/3.1)
 
-[Develop with Q# and Python](xref:microsoft.quantum.install.python) - Enables you to combine Python and Q# to create a Python host program that calls Q# operations.
+1. Install the `Microsoft.Quantum.IQSharp` package.
 
-[Develop with Q# and .NET](xref:microsoft.quantum.install.cs) - Combine C#, F#, or VB.NET with Q# to create a .NET host program that calls Q# operations.
+    ```dotnetcli
+        dotnet tool install -g Microsoft.Quantum.IQSharp
+        dotnet iqsharp install
+    ```
+
+To start writing Q# code with Jupyter Notebooks, see [Develop with Q# Jupyter Notebooks](xref:microsoft.quantum.install.jupyter).
```

Comment:
I think this line should be outside/below the tabbed UI, since it's not specific to the .NET CLI installation.